Makefile

```
@@ -1,60 +1,111 @@
 HOST=0.0.0.0
 PORT=8000
-PROJECT=control-panel
-MODULE=control_panel_api
+PROJECT=controlpanel
+MODULE=controlpanel
 VENV=venv
 BIN=${VENV}/bin
```

to make this work better for people who want to opt out of this Makefile creating a virtual env, this works better
```suggestion
BIN=${VIRTUAL_ENV}/bin
```